### ISSUE TITLE
Debianization

### ISSUE THREAD

**Issue by groolot** [aidanholm/luakit#383]
Hi Aidan,

I need a stable Debian package of luakit for my system and a liveUSB project I maintain for my students.

I would like to propose you to be the maintainer of this luakit version inside Debian. I'm totally new to this process but it seems that people is using luakit, but less than before (https://qa.debian.org/popcon.php?package=luakit).

This version based on WebKit2 is better than ever :)

Do you want me to help the project to achieve the Debianization?

**Comment by aidanholm** [comment]
Hey,
That sounds great; I've never maintained a Debian package and am unsure what that entails.
I'd be willing to merge any commits that help with that though!
Thanks :)

**Comment by groolot** [comment]
To do that, we have to work together on the same repository and I should have write access to the `debianization` branche only. We also will have to decide together the version name and release dates.

The other way is to package it from my fork where I will have a total control on version name and release. This is not my preference.

I think for example that the package could be named `luakit2` for two reasons:
 * the WebKit2GTK version is used and split compatibility with older version (say `2012.09.x`)
 * several great enhancements have been added by @aidanholm

Due to myjuvenile skills in this process, I'm trying to do it slowly and the most cleanly as I can.

**Comment by JR1994** [comment]
Please, anyway I can help let me know.

**Comment by aidanholm** [comment]
@groolot I don't believe GitHub supports restricting write access to a single branch. However, the original creator of luakit is offering to transfer ownership of luakit/luakit, and so this updated fork would be moved there and become the new mainline (#388). I'm not sure if this implies transfer of ownership of the luakit organization; if it does, then it should be straightforward to maintain a luakit/luakit-deb repo that would pull (automatically?) from luakit/luakit. That solves that problem.

I have in fact considered renaming my fork to luakit2, precisely because of the webkit version bump :) an unfork could mean just using the original `luakit` package name, however.

@JR1994 The best way to help is to open issues with any problems you find or features you'd find useful :) even "how do I do X" questions are great, because they indicate where the documentation is lacking.

**Comment by JR1994** [comment]
Will do!
